### ISSUE TITLE
Improve the accessibility of Acetate

### ISSUE THREAD

**Issue by SamReeve96** [SamReeve96/Acetate#17]
This should be implemented ASAP due to the importance of Accessibility. 

Research accessibility considerations and add to this issue anything that Acetate does not meet.

Issues:
* [ ] - Acetate does not allow users to select resizeable text
